img/leaderboard_plot.py

```diff
@@ -0,0 +1,42 @@
+import pandas as pd
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+# extract results from README
+results = {}
+readme = open('../README.md','r').read().split('\n')
+table_start = readme.index('|----|----:|----:|----:|----:|')
+table_end = readme.index('## Model')
+performance_table = readme[table_start+1: table_end-1]
+for record in performance_table:
+    model_name = record.split('|')[1].split()[-1]
```

Comment:
I think this assumes a format of: <company> <space> <model-name>
And I see how it works even if company is missing. But feels a bit fragile.
1. Do we want to list on the graph just the name of the model without company?
2. Could this break in the future if updating README is not according to these rules?

Reply:
> Do we want to list on the graph just the name of the model without company?
I discussed this with Forrest before, and he thought including only model names in the plot should be enough. The plot is to give an overview to the users. The detailed model information has been listed in the following section. If including company name is included in the plot, the names might be too long, making the plot unbalanced and not very good-looking.

> Could this break in the future if updating README is not according to these rules?
If we still follow the current update process (adding rows for new models to the table), it should work without problem. The changes only occur in the table section, and this part can be easily extracted. If the overall template will be changed in the future, then the code needs to be updated a bit, but it should also not be complex.